group_vars/location_mlk_nk/dns.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+dns_servers:
+  # quad9
+  - 9.9.9.9
```

Comment:
would be cool, to use `9.9.9.10` here instead of `9.9.9.9`. The latter one does implement DNS-blocking, which is not in line with our Expectations of a free network.

Similarly, the whole quad9-block should look like this:

```
- 9.9.9.10
- 149.112.112.10
- 2620:fe::10
- 2620:fe::fe:10
```
You can read this data at this website too:
https://quad9.org/service/service-addresses-and-features#unsec